clusterloader2/testing/density/config.yaml

```diff
@@ -94,20 +75,11 @@ steps:
       labelSelector: kind = latencyPod
       timeout: 10m
 - measurements:
-  - Identifier: PhaseTimer
-    Method: Timer
-    Params:
-      label: CreateLatencyPods
-      action: stop
   - Identifier: PodStartupLatency
     Method: PodStartupLatency
     Params:
       action: gather
-  - Identifier: PhaseTimer
-    Method: Timer
-    Params:
-      label: DeleteAllPods
-      action: start
+- name: Creating latency pods
 # Delete pods
```

Comment:
Please split the deletion of latency pods and saturation pods into two.

Also please add the timestamp after latency pods deletion.

Reply:
Done.